src/pyramid/config/security.py

```diff
@@ -236,9 +240,17 @@ def set_default_csrf_options(
         .. versionchanged:: 1.8
            Added the ``callback`` option.
 
+        .. versionchanged:: 2.0
+           Added the ``allow_no_origin`` option.
+
         """
         options = DefaultCSRFOptions(
-            require_csrf, token, header, safe_methods, callback
+            require_csrf=require_csrf,
```

Comment:
Not a huge fan of using keywords here for passing variables that are not keyword arguments in the target function... but I'll defer to @mmerickel for this.

Reply:
it's not a public api so it's a non-issue

On a side note re: Python 3 - I'm all for liberal use of keyword-only via `foo(*, arg1, arg2)` arguments in Pyramid 2.0!